lib/go-tc/federation_resolver.go

```diff
@@ -50,6 +51,36 @@ type FederationResolver struct {
 	TypeID      *uint      `json:"typeId,omitempty" db:"type"`
 }
 
+// [V5] FederationResolverV5 is an alias for the Federal Resolver struct response used for the latest minor version associated with api major version 5.
+type FederationResolverV5 = FederationResolverV50
+
+// [V50] FederationResolverV50 is used for RFC3339 format timestamp in FederationResolver which represents a resolver record for a CDN Federation.
+type FederationResolverV50 struct {
+	ID          *uint      `json:"id" db:"id"`
+	IPAddress   *string    `json:"ipAddress" db:"ip_address"`
+	LastUpdated *time.Time `json:"lastUpdated,omitempty" db:"last_updated"`
+	Type        *string    `json:"type"`
+	TypeID      *uint      `json:"typeId,omitempty" db:"type"`
+}
+
+// [V5] FederationResolversV5Response is an alias for the FederationResolvers struct response used for the latest minor version associated with api major version 5.
+type FederationResolversV5Response = FederationResolversV50Response
+
+// [V50] GET request to its /federation_resolvers endpoint for APIv5.
+type FederationResolversV50Response struct {
+	Alerts
+	Response []FederationResolverV5 `json:"response"`
+}
+
+// [V5] FederationResolverV5Response represents struct response used for the latest minor version associated with api major version 5.
+type FederationResolverV5Response = FederationResolverV50Response
+
+// [V50] POST request to its /federation_resolvers endpoint APIv5.
+type FederationResolverV50Response struct {
```

Comment:
Same comment here

Reply:
Also, following the conventions in our repo, the struct name should end with `V50`

Reply:
And here.